CppCoreGuidelines.md

```diff
@@ -13791,168 +13793,174 @@ Performance rule summary:
 * [Per.19: Access memory predictably](#Rper-access)
 * [Per.30: Avoid context switches on the critical path](#Rper-context)
 
-### <a name="Rper-reason"></a>Per.1: Don't optimize without reason
-
-##### Reason
-
-If there is no need for optimization, the main result of the effort will be more errors and higher maintenance costs.
+### <a name="Rper-reason"></a>Per.1: Understand your requirements and design accordingly.
 
-##### Note
+##### Reason
 
-Some people optimize out of habit or because it's fun.
+C++ is not the most common choice for situations with no performance demands at
+all, but performance demands are not equal to each other.  Before you can make
+any sensible choices you should understand what success looks like in your own
+domain.  Your requirements likely include some of binary size, latency,
+predictability, power, egress, and perhaps other more unique metrics.  What your
```

Comment:
Something is unique or it isn't, it can't be more unique.

Reply:
changed unique to exotic.

Reply:
Also thank  you for reading my suggestions and comments.

Reply:
> Everything is unique in the absolute sense but that is only interesting for a conversation over drinks. It isn't good writing.

I'm not going to argue with that :-)